amf-client/shared/src/test/resources/production/inter-spec-refs/reports/oas-raml-securityScheme.report

```diff
@@ -1,23 +1,32 @@
 Model: file://amf-client/shared/src/test/resources/production/inter-spec-refs/oas-raml-securityScheme/api.json
 Profile: OAS 2.0
 Conforms? false
-Number of results: 2
+Number of results: 3
```

Comment:
OasInterSpecRefsReportTest: this suite has two tests where an oas api references raml fragments. Previously the raml fragments where handled as external fragments (no parsing plugin matched), now the ExternalJsonYamlRefsPlugin parses the fragments which leads so some changes in the error messages.